Lib/json/tests/test_unicode.py

```diff
@@ -84,6 +84,19 @@ def test_bad_encoding(self):
         self.assertRaises(UnicodeEncodeError, self.loads, '"a"', u"rat\xe9")
         self.assertRaises(TypeError, self.loads, '"a"', 1)
 
+    def test_aliases(self):
+        self.assertEqual(type(self.dumps({"u": "t"}, ensure_ascii=False)), str)
+        self.assertEqual(type(self.dumps({"u": "t"}, ensure_ascii=False, encoding='u8')), str)
```

Comment:
AFAICS, https://docs.python.org/2/library/json.html?highlight=json#json.dump doesn't specify when it returns `str` and when `unicode`. Only that it _may_ return `unicode` if there are Unicode characters and/or `encoding` is specified.

So you can check that an alias returns the same type and value as the canonical name, but not what the type is.

To keep tests orthogonal, one also shouldn't needlessly assume that `utf-8` is the default. There's another test to check that already.